test.sh

```diff
@@ -2,8 +2,8 @@
 
 fails=0
 for i in $(grep -o 'travis.*\.svg' README.md); do
-  url=$(echo $i | sed "s/\//\/repositories\//" | sed "s/svg/json/")
-  curl -sq https://api.$url | grep 'last_build_result":0' > /dev/null
+  url=$(echo $i | sed "s/\//\/repositories\//" | sed "s/\.svg//")
+  curl -sq -H "Accept: application/json" https://api.$url | grep 'last_build_result":0' > /dev/null
```

Comment:
Can you explain what these changes do? We do have some failing plugins, but I was not aware of any issues in this code.

Reply:
sorry those changes were a misnomer